code/game/machinery/computer/ai_core.dm

```diff
@@ -9,14 +9,18 @@ var/global/list/empty_playable_ai_cores = list()
 	tool_interaction_flags = TOOL_INTERACTION_ALL
 	material = MAT_PLASTEEL
 
-	var/datum/ai_laws/laws = new /datum/ai_laws/nanotrasen
-	var/obj/item/stock_parts/circuitboard/circuit = null
-	var/obj/item/mmi/brain = null
+	var/datum/ai_laws/laws
+	var/obj/item/stock_parts/circuitboard/circuit
+	var/obj/item/mmi/brain
 	var/authorized
 
 	var/circuit_secured = FALSE
 	var/glass_installed = FALSE
 
+/obj/structure/AIcore/Initialize()
+	if(!laws)
+		laws = new GLOB.using_map.default_law_type
```

Comment:
Not calling `..()`
Not returning an initialization hint

Reply:
Added to the line below.